actions/image_partition_action.go

```diff
@@ -584,6 +586,61 @@ func (i ImagePartitionAction) PostMachineCleanup(context *debos.DebosContext) er
 	return nil
 }
 
+func ParseOffset(offset string) (int, string, error) {
+	/* Extract value of offset (int) and type from string*/
+	var v, t []rune
+	for _, l := range offset {
+		switch {
+		case l >= '0' && l <= '9':
+			v = append(v, l)
+		case l >= 'A' && l <= 'Z', l >= 'a' && l <= 'z', l == '%':
+			t = append(t, l)
+		}
+	}
```

Comment:
This loop here allows for arbitrary alpha-numeric word soup. For example "A01Z99%" is considered sane by the above.
Let's use a regex akin to the one in https://github.com/go-debos/debos/pull/292.

Reply:
@thanhtunng Any update?